README.rst

```diff
@@ -56,37 +78,68 @@ you'd want to check the following:
 - update the miniconda version and checksum
 
 Deployment
-----------
+~~~~~~~~~~
+#. Add the ansible-conda submodules, if they have not been installed already::
+
+    git submodule init 
+    git submodule update
 
-1. Rename the config file ``group_vars/jupyterhub_hosts.example`` to just
-   ``group_vars/jupyterhub_hosts``.
+#. Rename the config file ``group_vars/jupyterhub_hosts.example`` to just
+   ``group_vars/jupyterhub_hosts``::
+   
+    mv group_vars/jupyterhub_hosts.example group_vars/jupyterhub_hosts
 
-2. Run the following and paste the result into the config file under
+#. Run the following and paste the result into the ``jupyter_hosts`` config file under
    ``proxy_auth_token``::
 
     openssl rand -hex 32
```

Comment:
Should lines be indented after `::`?

Reply:
I think the lines should be indented after the `::`. It seemed as if many were not.